### DISCUSSION TITLE
Not really an issue, I just don't know what I'm doing

### DISCUSSION THREAD

**Discussion by Coenzyme480** [mxpv/podsync#267] [General]
After clicking "launch stack" and filling in the parameters, I get "CREATE_COMPLETE" on the AWS status overview.

But not sure what to do next. I see that I have a PodsyncUrl under outputs on AWS, but when I click on the url, to create an rss feed url, it says "This site can’t be reached ec2-3-........ refused to connect".

Thank you for creating this, any help is greatly appreciated!

**Comment by alessandroaime** [comment]
I've been running in the same issue lately.

I even went all the way to allowing all the possible types of inbound connections, as well as trying to access the RSS feed directly at `http://ec2.../ID1.xml` (without the `:8080` port reference as indicated in README).

@mxpv any idea what could cause it?

**Comment by JohannesHoppe** [comment]
I'm new to AWS, too. Could anybody explain to me what I should do next?
Under "Outputs" is an entry called `PodsyncUrl`, which points to `http://XXXX.eu-central-1.compute.amazonaws.com:8080/ID/`. But when I try to access that URL, it does not work:

> This site can’t be reached
> XXXX.eu-central-1.compute.amazonaws.com took too long to respond.

Any help would be highly appreciated! 🙏 


----

As far as I can see inbound rules where correctly set by the initial configuration, too!
<img width="1199" alt="Screenshot 2022-06-10 at 12 08 14" src="https://user-images.githubusercontent.com/640639/173043260-cd0fc413-0556-49dd-bea5-0801c45ad11e.png">

**Comment by JohannesHoppe** [comment]
This is some output from the EC2 Serial Console. Seems to be broken somehow..
<img width="813" alt="Screenshot 2022-06-10 at 12 19 14" src="https://user-images.githubusercontent.com/640639/173045262-1bc9bd49-8161-4dcb-b717-15c1dd8ef453.png">

Sorry, I have no clue what buttons I have to press...  🙈
